### ISSUE TITLE
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult

### ISSUE THREAD

**Issue by Rans0m00** [PowerShell/PSReadLine#2573]
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0-preview.6
OS: Microsoft Windows 10.0.21390
BufferWidth: 119
BufferHeight: 29

Last 0 Keys
```

```

```none
[run the script below and paste the output here]
```

## Exception report

```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004a6+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000478+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000f2+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000eb+0x60
```

## Steps to reproduce

open powershell

## Expected behavior

a useable terrminal

## Actual behavior

immediately printrs this and nothing else

**Comment by jbrown9100** [comment]
I received the same error the only difference is:

PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0-preview.7
OS: Microsoft Windows 10.0.21390
BufferWidth: 120
BufferHeight: 30

**Comment by SteveL-MSFT** [comment]
This is fixed in 2.2-beta3
